docs/daemon/cli.mdx

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 2
+sidebar_label: "CLI"
+id: daemon-cmdline
+title: Kwild CLI Guide
+description: Kwild Command Line
+slug: /daemon/cmdline
+---
+
+Kwild command starts both the blockchain and the database services of kwild node. 
+
+### Quickstart a kwild node:
+Run the below command to quickstart a kwild node without specifying any configuration.
+```bash
+kwild --autogen  --root_dir "path/to/root/dir"
```

Comment:
For quickstart, we probably should just do `kwild --autogen`, since there is a default root_dir.  Explaining `--root_dir` below how you did seems fine.

Reply:
Actually would it make sense to just save `--root_dir` for later in the docs, and not in the quickstart?